libs/components/packages/src/schematics/utility/visit-project-files.ts

```diff
@@ -0,0 +1,22 @@
+import { Tree } from '@angular-devkit/schematics';
+import { FileVisitor } from '@angular-devkit/schematics/src/tree/interface';
+
+const rootIgnore = ['dist', 'coverage'];
+const alwaysIgnore = ['node_modules'];
+
+export function visitProjectFiles(
+  tree: Tree,
+  projectPath: string,
+  visitor: FileVisitor,
+): void {
+  tree.getDir(projectPath).visit((path, entry) => {
+    if (
+      path.includes('/.') ||
```

Comment:
Is this matching filenames that start with `.`? Or relative paths? e.g. `../../../foobar`

Reply:
Filenames that start with `.`

Reply:
Wouldn't `../../../foobar` also be matched with this change?

Reply:
The `Tree` `visit` method only walks down the tree, so these wouldn't matter.